src/Client/Pages/Identity/Users/Users.razor.cs

```diff
@@ -34,10 +34,17 @@ public partial class Users
 
     protected override async Task OnInitializedAsync()
     {
+        const string CLAIMTYPE_ENDSWITH = "claims/name";
+
         var user = (await AuthState).User;
         _canExportUsers = await AuthService.HasPermissionAsync(user, FSHAction.Export, FSHResource.Users);
         _canViewRoles = await AuthService.HasPermissionAsync(user, FSHAction.View, FSHResource.UserRoles);
 
+        string? tennant = user.Claims
+                            .Where(u => u.Type.EndsWith(CLAIMTYPE_ENDSWITH))
+                            .Select(x => x.Value)
+                            .FirstOrDefault() ?? string.Empty;
```

Comment:
You can simply use user.GetTenant() (from Shared/authorization/claimsprincipalextensions)